hackclub/hcb
# First-time activation for cards

Part of #2994 

## to test
- manually issue a replacement card through the Stripe API or CLI
- import the replacement through the Rails console (see #3032 for an example)
- test it out
